nss-day-cohort-51/nss-day-cohort-51.github.io
# Mw dev card remove video icon 126

# Description
#118 Updated Modal with functions that shows default image for capstones without videos or a github repo image. Linked the youtube video if the user's capstone link is from youtube. 
#119 Change modal title to **Capstones**
#126 Removed the video icon 

## Type of change


# Testing Instructions

- Fetch branch and run npm start to serve the website in a browser
- On all student cards, the Learn More button should not have a video icon
- Click Learn More button to open modal
- Observe the title about the capstone section reads **Capstones**
- Check that the Capstone title is a link that goes to the capstone repo on github; check that the capstone image(if applicable) is visible and links to the capstone repo;  


ADDITION:
Added a "More Details" button on the Event Card.
Added styling to fall in-line with existing buttons.

# Checklist:
